lib/base_service.js

```diff
@@ -390,10 +389,16 @@ class BaseService {
                 config.logging.name = name;
             }
 
-            // And the statsd client
+            // And metrics clients
             if (!config.metrics.name) {
                 config.metrics.name = name;
             }
+            if (!config.metrics.prometheus) {
+                config.metrics.prometheus = {};
+            }
+            if (!config.metrics.prometheus.name) {
+                config.metrics.prometheus.name = name;
+            }
```

Comment:
Should we use a more confine `config.blabla = config.blabla || foobar`?

Reply:
I took a shot at making that more concise.  Let me know what you think